docker/install/ubuntu_install_onnx.sh

```diff
@@ -21,7 +21,7 @@ set -u
 set -o pipefail
 
 # fix to certain version for now
-pip3 install onnx==1.5.0
+pip3 install onnx==1.6.0
 pip3 install onnxruntime==1.0.0
 
 # torch depends on a number of other packages, but unhelpfully, does
```

Comment:
have you updated torch?

Reply:
nice catch, the binary image is correct, somehow i missed the patch.
```
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
i>>> import torchvision
>>> torch.__version__
'1.4.0'
>>> torchvision.__version__
'0.5.0'
>>> 
```

Reply:
ok